File: Proyecto Riojas/Proyecto Final1/Proyecto Final1/Estructura de datos .cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;

namespace Proyecto_Final1
{
    public partial class Estructura_de_datos : Form
    {
        public Estructura_de_datos()
        {
            InitializeComponent();

        }

        private void btnLiDE_Click(object sender, EventArgs e)
        {
            ListasDEnlazadas LDE = new ListasDEnlazadas();
            LDE.Show();
            
        }

        private void btnLiS_Click(object sender, EventArgs e)
        {
            ListasSimples LS = new ListasSimples();
            LS.Show();
        }

        private void btnLiC_Click(object sender, EventArgs e)
        {
            ListasCirculares LC = new ListasCirculares();
            LC.Show();
        }

        private void btnLiDC_Click(object sender, EventArgs e)
        {
            ListasDCirculares LDC = new ListasDCirculares();
            LDC.Show();
        }

        private void btnPilas_Click(object sender, EventArgs e)
        {
            Pilas P = new Pilas();
            P.Show();
        }

        private void btnColas_Click(object sender, EventArgs e)
        {
            Colas C = new Colas();
            C.Show();
        }

        private void btnArboles_Click(object sender, EventArgs e)
        {
            Arboles A = new Arboles();
            A.Show();
        }

        private void BtnGrafos_Click(object sender, EventArgs e)
        {
            Grafos G = new Grafos();
            G.Show();
        }

    private void Estructura_de_datos_Load(object sender, EventArgs e)
    {

    }
  }
}

File: ProyectovisualBasic/ProyectovisualBasic/EstructuraDeDatos.vb
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Linq
Imports System.Text
Imports System.Threading.Tasks
Imports System.Windows.Forms

Namespace Proyecto_Final1

    Public Class Estructura_de_datos
        Inherits Form

        Public Sub New()
            MyBase.New
            InitializeComponent
        End Sub

        Private Sub btnLiDE_Click(ByVal sender As Object, ByVal e As EventArgs)
            Dim LDE As ListasDEnlazadas = New ListasDEnlazadas
            LDE.Show
        End Sub

        Private Sub btnLiS_Click(ByVal sender As Object, ByVal e As EventArgs)
            Dim LS As ListasSimples = New ListasSimples
            LS.Show
        End Sub

        Private Sub btnLiC_Click(ByVal sender As Object, ByVal e As EventArgs)
            Dim LC As ListasCirculares = New ListasCirculares
            LC.Show
        End Sub

        Private Sub btnLiDC_Click(ByVal sender As Object, ByVal e As EventArgs)
            Dim LDC As ListasDCirculares = New ListasDCirculares
            LDC.Show
        End Sub

        Private Sub btnPilas_Click(ByVal sender As Object, ByVal e As EventArgs)
            Dim P As Pilas = New Pilas
            P.Show
        End Sub

        Private Sub btnColas_Click(ByVal sender As Object, ByVal e As EventArgs)
            Dim C As Colas = New Colas
            C.Show
        End Sub

        Private Sub btnArboles_Click(ByVal sender As Object, ByVal e As EventArgs)
            Dim A As Arboles = New Arboles
            A.Show
        End Sub

        Private Sub BtnGrafos_Click(ByVal sender As Object, ByVal e As EventArgs)
            Dim G As Grafos = New Grafos
            G.Show()
        End Sub

        Private Sub Estructura_de_datos_Load(ByVal sender As Object, ByVal e As EventArgs)

        End Sub

        Private Sub InitializeComponent()
            Me.SuspendLayout()
            '
            'Estructura_de_datos
            '
            Me.ClientSize = New System.Drawing.Size(278, 244)
            Me.Name = "Estructura_de_datos"
            Me.ResumeLayout(False)

        End Sub

        Private Sub Estructura_de_datos_Load_1(sender As Object, e As EventArgs) Handles MyBase.Load

        End Sub
    End Class
End Namespace